frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-transfer-counts/radiogram-card-content-transfer-counts.component.ts

```diff
@@ -0,0 +1,29 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type { TransferCountsRadiogram } from 'digital-fuesim-manv-shared';
+import { UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import { createSelectRadiogram } from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-radiogram-card-content-transfer-counts',
+    templateUrl: './radiogram-card-content-transfer-counts.component.html',
+    styleUrls: ['./radiogram-card-content-transfer-counts.component.scss'],
+})
+export class RadiogramCardContentTransferCountsComponent implements OnInit {
+    readonly categories = ['red', 'yellow', 'green'] as const;
```

Comment:
Please also display the other categories. It would be nice if the three are always displayed and the others only if not empty

Reply:
I've enabled showing all triage categories in 8cb457e5